app/api/v1/models/questions_model.py

```diff
@@ -0,0 +1,59 @@
+""" Manages all the meetup functions """
+from datetime import date
+from flask import Flask, request, jsonify
+from app.api.v1.utils.user_validator import UsersHelper
+
+
+class QuestionsModel():
+
+    def __init__(self):
+        self.questions = []
+        self.helpers = UsersHelper()
+
+    def add_a_question(self, user_request):
+        """ Add a question record """
+        question = {}
+        keys_expected = ["user", "meetup", "title", "body"]
+
+        ret = self.helpers.is_valid_user_request(keys_expected, user_request)
+
+        if ret[0] == 0:
+            return ret[1], 400
+
+        ret = self.helpers.is_blank_field(user_request)
+
+        if ret[0] == 0:
+            return ret[1], 400
+        question['id'] = len(self.questions) + 1
+        question['votes'] = 0
+        question['user'] = user_request.get_json()['user']
+        question['meetup'] = user_request.get_json()['meetup']
+        question['title'] = user_request.get_json()['title']
+        question['body'] = user_request.get_json()['body']
+
+        self.questions.append(question)
+        return jsonify({"msg": "question was added", "data":self.questions, "status": 201}), 201
+
+    def vote(self, vote, question_id, type):
+        """ Allows a user to upvote or downvote a question """
+
+        question = [q for q in self.questions if q["id"] == question_id]
```

Comment:
use descriptive variable names as opposed to `q`

Reply:
I've Implemented the changes recommended on branch name bg-fix-failing-build-163348045